readme/clipper.md

```diff
@@ -1,9 +1,37 @@
 # Joplin Web Clipper
 
-The Web Clipper is a browser extension that allows you to save web pages and screenshots from your browser. To start using it, open the Joplin desktop application, go to the **Web Clipper Options** and follow the instructions.
+The Web Clipper is a browser extension that allows you to save web pages and screenshots from your browser. Currently, the Web Clipper is available as a browser extension for Google Chrome as well as Mozilla Firefox. To start using it, open the Joplin desktop application, go to the **Web Clipper Options** and follow the instructions.
 
 <img src="https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/WebsiteAssets/images/WebExtensionScreenshot.png" style="max-width: 50%; border: 1px solid gray;">
 
+# Installation and Setup
+
+- Install the Joplin Web Clipper browser extension for either [Google Chrome](https://chrome.google.com/webstore/detail/joplin-web-clipper/alofnhikmmkdbbbgpnglcpdollgjjfek) or [Mozilla Firefox](https://addons.mozilla.org/en-US/firefox/addon/joplin-web-clipper/).
+- Install the Joplin Desktop application, if you have not already done so.
+- Open the Joplin Desktop app and select the **Tools** tab at the top.
+- Then, click on the **Options** tab from the dropdown menu.
```

Comment:
Replace this with `- Open the [Configuration screen](https://github.com/laurent22/joplin/blob/dev/readme/config_screen.md)`